### ISSUE TITLE
ETL user facilities submission metadata transformation completed for JGI and EMSL

### ISSUE THREAD

**Issue by mslarae13** [microbiomedata/issues#238]
Complete when these are complete 

- [x] https://github.com/microbiomedata/metadata-for-user-facility-template-transformations/issues/5
- [x] https://github.com/microbiomedata/metadata-for-user-facility-template-transformations/issues/19

**Comment by mslarae13** [comment]
Blocked, I need to provide Sujay a term and class map

**Comment by mslarae13** [comment]
See https://github.com/microbiomedata/metadata-template-transformation/tree/issue238_MUTTs_term-map-update

**Comment by ssarrafan** [comment]
@mslarae13 @sujaypatil96 is this still blocked?  Backlog?

**Comment by ssarrafan** [comment]
I haven't heard back on this issue so I'm going to remove it from the sprint and add the backlog label so I can close sprint 10.

**Comment by sujaypatil96** [comment]
No, the issue isn't blocked. The script is ready for the most part, I'm waiting for a quick review on the output that the script produced for one of the test studies on the submission portal from @mslarae13

**Comment by ssarrafan** [comment]
> No, the issue isn't blocked. The script is ready for the most part, I'm waiting for a quick review on the output that the script produced for one of the test studies on the submission portal from @mslarae13

@sujaypatil96 should I assign this to you for this sprint?

**Comment by sujaypatil96** [comment]
Yup, you can assign this to me for this sprint. I'm mostly done working on this, so it should be closed out this sprint.

**Comment by mslarae13** [comment]
See https://github.com/microbiomedata/metadata-template-transformation/issues/5 for requested updates

**Comment by ssarrafan** [comment]
@sujaypatil96 what's the status of this issue?

**Comment by ssarrafan** [comment]
Montana said this is in progress. Moving to new sprint.

**Comment by ssarrafan** [comment]
@sujaypatil96 @mslarae13 move to next sprint again or backlog?

**Comment by sujaypatil96** [comment]
@ssarrafan backlog for now. I'm focussing on NEON data ingest pipeline this sprint.

**Comment by mslarae13** [comment]
I've at @cristina-stonepedraza to pick this issue up next sprint

**Comment by mslarae13** [comment]
@cristina-stonepedraza is working on JGI
@Natalie-Winans is working on EMSL

**Comment by ssarrafan** [comment]
@mslarae13 should this be added to this sprint?
